src/newcharacter.cpp

```diff
@@ -1764,12 +1764,12 @@ int set_description(WINDOW *w, player *u, character_type type, int &points)
 
         werase(w_scenario);
         mvwprintz(w_scenario, 0, 0, COL_HEADER, _("Scenario: "));
-        wprintz(w_scenario, c_ltgray, g->scen->gender_appropriate_name(u->male).c_str());
+        wprintz(w_scenario, c_ltgray, _(g->scen->gender_appropriate_name(u->male).c_str()));
         wrefresh(w_scenario);
 
         werase(w_profession);
         mvwprintz(w_profession, 0, 0, COL_HEADER, _("Profession: "));
-        wprintz (w_profession, c_ltgray, u->prof->gender_appropriate_name(u->male).c_str());
+        wprintz (w_profession, c_ltgray, _(u->prof->gender_appropriate_name(u->male).c_str()));
```

Comment:
Are you sure this is required? It seems the names are already translated via `pgettext` in `profession::load_profession`.